pilot/pkg/status/state.go

```diff
@@ -56,54 +59,56 @@ type DistributionController struct {
 	CurrentState     map[Resource]map[string]Progress
 	ObservationTime  map[string]time.Time
 	UpdateInterval   time.Duration
```

Comment:
This is not used anymore?

Reply:
no, still used. Just set default value in the construction funtion